packages/frontend/src/hooks/useEcharts.ts

```diff
@@ -142,7 +142,9 @@ export const getEchartsSeries = (
     originalData: ApiQueryResults['rows'],
     cartesianChart: CartesianChart,
     pivotKey: string | undefined,
-    formats: Record<string, CompiledField> | undefined,
+    formats:
```

Comment:
:see_no_evil:

Reply:
Haha it's ok we have this all over the codebase. But it makes it much more flexible if we only pick what we need